src/index.html

```diff
@@ -43,27 +43,36 @@
         </nav>
     </header>
     </div>
-    <div >
+
+
+    <div>
         <div class="row">
             <form class="form-horizontal container">
-                <label class="col-md-10">
-                <div  class="form-group">
-                    <input type="file" accept=".json" id="myJsonFile" class="control-label col-xs-4" multiple>
+                <div class="form-group">
+                    <label class="col-md-10">
+                        <input type="file" accept=".json" id="myJsonFile" class="control-label col-xs-4" multiple>
                     </label>
-                    </div>
-                    <input type="submit" class="btn btn-success" value="Upload File" ng-click="uploader()">
-                    <input type="submit" value="Create Index" class="btn btn-success" ng-click="submitIndex()">
-                    <br><br>
-                    </form>
-                    <div>{{ uploaderError }}</div>
-                
+                </div>
+                <input type="submit" class="btn btn-success" value="Upload File" ng-click="uploader()">
+                <input type="submit" value="Create Index" class="btn btn-success" ng-click="submitIndex()">
                 <br><br>
```

Comment:
You might want to avoid using <br> tag in your html
You can use css for that `margin-top` or `margin-bottom` property should handle that